source/text/Font.cpp

```diff
@@ -127,19 +142,29 @@ void Font::DrawAliased(const DisplayText &text, double x, double y, const Color
 
 
 
-void Font::Draw(const string &str, const Point &point, const Color &color) const
+void Font::Draw(const string &str, const Point &point, const Color &color, bool matchBack) const
 {
-	DrawAliased(str, round(point.X()), round(point.Y()), color);
+	DrawAliased(str, round(point.X()), round(point.Y()), color, matchBack);
 }
 
 
 
-void Font::DrawAliased(const string &str, double x, double y, const Color &color) const
+void Font::DrawAliased(const string &str, double x, double y, const Color &color, bool matchBack) const
 {
 	glUseProgram(shader.Object());
+	glUniform1i(texI, 0);
+	glUniform1i(backgroundI, 1);
+	glActiveTexture(GL_TEXTURE0 + 0);
 	glBindTexture(GL_TEXTURE_2D, texture);
+	if(matchBack)
+	{
+		glActiveTexture(GL_TEXTURE0 + 1);
```

Comment:
GL_TEXTURE1 Exists

Reply:
I know. I personally prefer this don't ask me why. I do think I should use GL_TEXTURE1